gradle.properties

```diff
@@ -17,3 +17,5 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+# ID query parameters to call the API
+APP_ID="b6243258fce26de557248d662c2aef4c"
```

Comment:
the idea is to not have this in the project, please put this in the global properties file so it will keep only on your computer and not in the repo

Reply:
Done